htdocs/comm/action/class/cactioncomm.class.php

```diff
@@ -169,8 +169,8 @@ public function liste_array($active = '', $idorcode = 'id', $excludetype = '', $
                         if ($obj->module == 'invoice' && ! $conf->facture->enabled)	 $qualified=0;
                         if ($obj->module == 'order'   && ! $conf->commande->enabled) $qualified=0;
                         if ($obj->module == 'propal'  && ! $conf->propal->enabled)	 $qualified=0;
-                        if ($obj->module == 'invoice_supplier' && ! $conf->fournisseur->enabled)   $qualified=0;
-                        if ($obj->module == 'order_supplier'   && ! $conf->fournisseur->enabled)   $qualified=0;
+                        if ($obj->module == 'invoice_supplier' && ((! $conf->fournisseur->enabled && $conf->global->MAIN_USE_NEW_SUPPLIERMOD) || ! $conf->supplier_invoice->enabled))	$qualified=0;
```

Comment:
Are you sure this is not instead 
if ($obj->module == 'invoice_supplier' && ((! $conf->fournisseur->enabled && empty($conf->global->MAIN_USE_NEW_SUPPLIERMOD)) || ! $conf->supplier_invoice->enabled))	$qualified=0;

Reply:
You right, it's fix now.